protocol/operation.re

```diff
@@ -52,10 +52,16 @@ module Main_chain = {
 
 module Side_chain = {
   // TODO: I don't like this structure model
+  [@deriving yojson]
+  type timestamp = float;
+
   [@deriving yojson]
   type kind =
     | Transaction({destination: Wallet.t})
-    | Withdraw({owner: Tezos_interop.Address.t});
+    | Withdraw({owner: Tezos_interop.Address.t})
+    | Add_validator(Validators.validator, timestamp)
```

Comment:
This should not have a timestamp

Reply:
Addressed in the next commit by removing the timestamp from the operation.